src/openms_gui/source/VISUAL/DIALOGS/SwathTabWidget.cpp

```diff
@@ -321,87 +341,103 @@ namespace OpenMS
       ui->input_swath_windows->setCWD(new_cwd);
     }
 
+    /// custom arguments to allow for looping calls
```

Comment:
what is this looping thing about ;)
can you add one or two more lines of documentation to COmmand and Args